configs/apollodata.json

```diff
@@ -4,17 +4,23 @@
     "https://www.apollographql.com/docs/"
   ],
   "stop_urls": [
-    "index\\.html"
+    "#"
   ],
   "selectors": {
-    "lvl0": ".content h1",
-    "lvl1": ".content h2",
-    "lvl2": ".content h3",
-    "lvl3": ".content h4",
-    "lvl4": ".content h5",
-    "text": ".content p, .content li"
+    "default": {
+      "lvl0": {
+        "selector": "div.sidebar span.title-sidebar",
```

Comment:
The goal here is to have each docs repository automatically appear in a group based on its title.  The title is available on the sidebar, inside the `title-sidebar` `span` tag.